src/app/routes/connector-ui/transfer-history-page/transfer-history-page/transfer-history-page.component.html

```diff
@@ -36,51 +36,48 @@
       class="transfer-history-table"
       mat-table
       [dataSource]="transferProcessesList.data.transferProcesses">
-      <ng-container matColumnDef="id">
-        <th *matHeaderCellDef mat-header-cell scope="col">ID</th>
-        <td *matCellDef="let item" mat-cell>{{ item.id }}</td>
+      <ng-container matColumnDef="direction">
+        <th *matHeaderCellDef mat-header-cell scope="col">Direction</th>
+        <td *matCellDef="let item" mat-cell>
+          <mat-icon mat-table-avatar>
+            {{ item.direction === 'PROVIDING' ? 'upload' : 'download' }}
+          </mat-icon>
+        </td>
       </ng-container>
-
-      <ng-container matColumnDef="creationDate">
-        <th *matHeaderCellDef mat-header-cell scope="col">Created</th>
+      <ng-container matColumnDef="assetId">
+        <th *matHeaderCellDef mat-header-cell scope="col">Asset</th>
         <td *matCellDef="let item" mat-cell>
-          {{ asDate(item.createdAt) }}
+          <div *ngIf="item.direction === 'PROVIDING'">
+            <a
+              class="link"
+              href="javascript:void(0)"
+              (click)="onAssetDetailsClick(item.transferProcessId)">
+              {{ item.assetName ?? item.assetId }}
+            </a>
+          </div>
+          <div *ngIf="item.direction === 'CONSUMING'">
+            {{ item.assetName ?? item.assetId }}
+          </div>
         </td>
       </ng-container>
-
       <ng-container matColumnDef="state">
         <th *matHeaderCellDef mat-header-cell scope="col">State</th>
-        <td *matCellDef="let item" mat-cell>{{ item.state }}</td>
+        <td *matCellDef="let item" mat-cell>{{ item.state.name }}</td>
```

Comment:
in the contract agreement page we use the state's "meta state" (e.g. in progress, error, success) to add highlights, we should do the same here

(i think it's a loading icon and a warning icon with the details or sth)